src/main/java/org/pesmypetcare/webservice/service/MedicationServiceImp.java

```diff
@@ -0,0 +1,64 @@
+package org.pesmypetcare.webservice.service;
+
+import org.pesmypetcare.webservice.dao.MedicationDao;
+import org.pesmypetcare.webservice.entity.MedicationEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+@Service
+public class MedicationServiceImp implements MedicationService {
```

Comment:
I think that this document has to have the "Impl" termination despite Imp.

Reply:
Done